RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
```

Comment:
As per discussion in comments, I think this needs to be a minor release rather than a patch.